yaml2ncml/meta.yaml

```diff
@@ -1,10 +1,10 @@
 package:
     name: yaml2ncml
-    version: "0.1.0"
+    version: "0.2.0"
 
 source:
-    git_url: https://github.com/rsignell-usgs/yaml2ncml.git
-    git_tag: v0.1.0
+    git_url: https://github.com/usgs-cmg/yaml2ncml.git
+    git_tag: v0.2.0
```

Comment:
Your git tag is `0.2.0` instead of `v0.2.0`.  My git-fu is not good enough to rebase and fix this, because this is not a fork, so I am re-submitting this PR (See #315).